ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/QueuedCommand.java

```diff
@@ -39,6 +39,10 @@ public QueuedCommand(final CommandId commandId,
     this(commandId, command, Optional.empty(), -1L);
   }
 
+  QueuedCommand(final CommandId commandId, final Command command, final long offset) {
```

Comment:
Too many constructors!

Think about removing the old package private constructor in favour of your new one.  

Also, could you remove the old public constructor below - it's does not seem to be used.